linera-client/src/unit_tests/mod.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(test)]
+mod util;
+
+#[cfg(test)]
+mod chain_listener;
+
+#[cfg(test)]
+mod wallet;
```

Comment:
Isn't the whole module already `cfg(test)`?

```suggestion
mod util;
mod chain_listener;
mod wallet;
```

Reply:
Good point 😅 Will fix